### ISSUE TITLE
Enable RDP in deployment link doesn't enable RDP

### ISSUE THREAD

**Issue by silversword411** [amidaware/tacticalrmm#1436]
**Server Info (please complete the following information):**
 - OS: Debian 11
 - RMM Version (as shown in top left of web UI): 0.15.7

**Installation Method:**
  - [x] Standard

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 2.4.4
- Agent OS: Windows 11 Pro, 64 bit v22H2 (build 22621.1194)

**Describe the bug**
Enable RDP...doesn't enable RDP

<img width="281" alt="2023-02-14_142424 - trmm doesnt enable rdp" src="https://user-images.githubusercontent.com/13395348/218839788-647b9d49-7ab3-43a8-99a2-9d16f3f19cdd.png">

<img width="511" alt="2023-02-14_143917 - win11pro RDP not enabled" src="https://user-images.githubusercontent.com/13395348/218839975-3b706d97-181e-4bee-83c4-183248fe9975.png">

**Comment by styx-tdo** [comment]
Hi,
Linux does not feature "RDP", only remote control via Mesh Agent. Also, that only works when a GUI is installed (XWindows, maybe Wayland)

For command line access, use "remote background"

Or do I miss something here?

**Comment by silversword411** [comment]
RDP (Remote Desktop Protocol) is windows (not home) only.

I believe that checkbox is there, and designed to enable the RDP service and enable firewall rules automatically when TRMM agent installs. 

I'm saying it doesn't do that.

**Comment by wdarney** [comment]
Yeah, if you choose linux, RDP doesn't even show up. So this is clearly for windows. I too have the same issue. I just enable it manually via powershell after tactical gets installed.

**Comment by silversword411** [comment]
> I just enable it manually via powershell after tactical gets installed

You have a script for that, would be good to have an RDP toggle in https://github.com/amidaware/community-scripts
